catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/InspectCommand.java

```diff
@@ -79,12 +81,14 @@ protected Object executeWithSubject() throws Exception {
 
       for (AttributeDescriptor ad : mType.getAttributeDescriptors()) {
 
+        boolean isMultiValued = ad.isMultiValued();
+
         Attribute attribute = card.getAttribute(ad.getName());
 
-        Serializable value = null;
+        List<Serializable> value = null;
 
         if (attribute != null) {
-          value = attribute.getValue();
+          value = isMultiValued ? attribute.getValues() : ImmutableList.of(attribute.getValue());
```

Comment:
:question: Since you used an ImmutableList for attribute.getValue() do you want to use one for attribute.getValues()?

Reply:
Yeah I'll add that in as well